src/constant_time.rs

```diff
@@ -33,5 +33,5 @@ pub fn verify_slices_are_equal(a: &[u8], b: &[u8])
 }
 
 extern {
-    fn GFp_memcmp(a: *const u8, b: *const u8, len: c::size_t) -> c::int;
+    pub fn GFp_memcmp(a: *const u8, b: *const u8, len: c::size_t) -> c::int;
```

Comment:
It is, at least as of `rustc` 1.15.1. I would only expect it to be unnecessary if `constant_time` were a parent module of `ec::dsa`, but it is only an aunt(?).